advanced_html_css/accessibility/wai_aria.md

```diff
@@ -58,29 +58,29 @@ When using such ARIA attributes, you would provide an `id` to one element, and y
 
 #### `aria-label`
 
-The `aria-label` attribute overrides any native label and modifies the name property in the accessibility tree, though it's best used when an element doesn't already have a native label. When you add `aria-label` to an element, you pass in a string as the value, which will become that elements accessible name. `aria-label` doesn't work on every HTML element, though. Adding the attribute to a plain `<div>` or a `<span>` will have no effect, for example.
+The `aria-label` attribute overrides any native label of an element and modifies its name property in the accessibility tree. It is best used when an element doesn't already have a native label. When added, the string value of `aria-label` attribute  becomes the element's accessible name. **However**, [`aria-label` does not have any effect on some HTML elements](https://github.com/w3c/aria/issues/756), such as `<div>` or a `<span>`. 
```

Comment:
```suggestion
The `aria-label` attribute overrides any native label of an element and modifies its name property in the accessibility tree. It is best used when an element doesn't already have a native label. When added, the string value of the `aria-label` attribute  becomes the element's accessible name. **However**, [`aria-label` does not have any effect on some HTML elements](https://github.com/w3c/aria/issues/756), such as `<div>` or a `<span>`. 
```

For the last sentence, I think this could be a good use-case for our note boxes, but not really a blocker. @Lofty-Brambles I'm not entirely sure about linking to the W3C issue here... Also not really a blocker, but could we instead just mention the same info ("'aria-label' may not have any - or the intended - effect on certain HTML elements, such as a `div` or `span` without a `role` attribute.")?

Reply:
Sure thing! I found the conversation interesting, since it came from the very same confusion, which's why I linked it here. Not a blocker if you think it is not important. @thatblindgeye

Reply:
I think my main worry is whether it will bog users down on a conversation that they may not understand the context of (could just be an invalid worry 😆 ). I don't really feel super strong either way; worse case is we keep the link and if any users start questioning it we can replace it in the future.

Reply:
I do not feel so strongly about whether to keep the link or not either. I think the learners will know not to spend too much time understanding the reasoning behind this. If they *want* to dive deeper into the issue, they've got their source.